TFG-Labs/custom-shipping-step-by-items
# Feature/tlva 125/check furniture masterdata

### What problem is this solving?

Functionality developed to satisfy customer requirements for furniture products requiring additional information

#### How it works

If there is a product from the furniture category in the shopping cart, the customer will be shown a form that must be filled in.

The data from this form will be stored with the address in Masterdata.

#### How to test it?

[Workspace](https://tlva125--thefoschini.myvtex.com/)

You must be logged in with a customer already registered on the platform.

In the checkout add the product with SKU **59418098 or 59418103 **, which has category "furniture".

When you reach the Shipping step and you have entered the shipping address, the form will appear as you can see in the following video

### Screenshots/Video example usage

[Video](https://www.loom.com/share/eb2da5471028402aa052015d8459d276)

#### How does this PR make you feel? 🔗

![](https://media.giphy.com/media/3oEdv1EbS2Ss1NvrUc/giphy.gif)
